googleServiceAccount.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We need to have this ignored by git. I will create a new one

Reply:
We also need this in the authenticate middleware

`  if (process.env.NODE_ENV === 'development') return next(); `
  
To skip authentication in development